easybuild/framework/easyconfig/types.py

```diff
@@ -400,6 +400,12 @@ def to_dependencies(dep_list):
     """
     return [to_dependency(dep) for dep in dep_list]
 
+def to_checksums(checksums):
+    if not isinstance(checksums[0], list):
```

Comment:
I'm not sure why this is needed either, it looks wrong to only consider the 1st element.